.travis.yml

```diff
@@ -14,6 +14,11 @@ before_install:
 addons:
   chrome: stable
 
+before_script:
+  - LATEST_CHROMEDRIVER_VERSION=`curl -s "https://chromedriver.storage.googleapis.com/LATEST_RELEASE"`
```

Comment:
Where is LATEST_CHROMEDRIVER_VERSION used?

Reply:
In the next line :)

Reply:
How does it sync with the changes in wdio.conf.js?

Reply:
For now, I removed it since I'm not sure if it works properly. Will check it next time.
Thanks!